rust/src/cells/reimplemented.rs

```diff
@@ -344,16 +344,13 @@ pub const fn cell_wide_left_p(cell: &NcCell) -> bool {
 /// *Method: NcCell.[load_char()][NcCell#method.load_char].*
 //
 // NOTE: Unlike the original C function this doesn't return anything.
-// REMINDER: remove casting for NCCELL_WIDEASIAN_MASK when fixed:
 // Waiting for: https://github.com/rust-lang/rust-bindgen/issues/1875
 #[inline]
 pub fn cell_load_char(plane: &mut NcPlane, cell: &mut NcCell, ch: NcEgc) /* -> i32 */
 {
     unsafe {
         crate::cell_release(plane, cell);
     }
```

Comment:
Don't we need something here to load the char?

Reply:
yes, regardless of the `CELL_ASIAN_MASK` value